FlyWireAnnotationTutorial.ipynb

```diff
@@ -0,0 +1,309 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The CAVEclient\n",
+    "\n",
+    "The CAVEclient is a client side library to allow easy interaction with the services within CAVE (connectome annotation versioning engine, also known as Dynamic Annotation Framework), eg. the annotations, stateserver. The github repository is public:\n",
+    "https://github.com/seung-lab/CAVEclient\n",
+    "\n",
+    "The library can be installed directly from the github repository or from the prebuilt versions using pip:\n",
+    "```\n",
+    "pip install caveclient\n",
+    "```\n",
+    "\n",
+    "\n",
+    "## Tutorials\n",
+    "\n",
+    "This tutorial mainly covers the interactions with the materialized annotation tables. More information and better explanations of the other functionalities of the client can be found in the following tutorial. Please be advised that depending on your permission level you may not be able to execute all queries in this tutorial with the preset parameters as it was written with defaults for iarpa's microns project:\n",
+    "https://github.com/seung-lab/CAVEclient/blob/master/CAVEclientExamples.ipynb\n",
+    "\n",
+    "\n",
+    "## Authentication & Authorization\n",
+    "\n",
+    "If this is your first time to interact with any part of CAVE, chances are you need to setup your local credentials for your FlyWire account first. Please follow the section \"Setting up your credentials\" at the beginning of the tutorial above to do so.\n",
+    "\n",
+    "You will need to have access to the FlyWire's production dataset to retrieve annotations. Otherwise you will see\n",
+    "\n",
+    "```HTTPError: 403 Client Error: FORBIDDEN for url```\n",
+    "\n",
```

Comment:
This suggests on one line you need microns permissions, and the next that you need flywire permissssions

Reply:
I don't quite follow. There is just a disclaimer about the linked notebook
```
Please be advised that depending on your permission level you may not be able to execute all queries in this tutorial with the preset parameters as it was written with defaults for iarpa's microns project:
```